dotnetv3/Lambda/CreateDynamoDBTableExample/CreateDynamoDBTableExample/Function.cs

```diff
@@ -0,0 +1,103 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.RuntimeSupport;
+using Amazon.Lambda.Serialization.SystemTextJson;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// This project specifies the serializer used to convert Lambda event into .NET classes in the project's main 
+// main function. This assembly register a serializer for use when the project is being debugged using the
+// AWS .NET Mock Lambda Test Tool.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace CreateDynamoDBTableExample
+{
+    public class Function
+    {
+        /// <summary>
```

Comment:
Isn't this stating the obvious?

Reply:
It certainly is. That's what I get for being lazy and leaving the comments from the project template.